src/createAsyncThunk.ts

```diff
@@ -223,6 +223,13 @@ export interface AsyncThunkOptions<
   dispatchConditionRejection?: boolean
 
   serializeError?: (x: unknown) => GetSerializedErrorType<ThunkApiConfig>
+
+  /**
+   * A function to use when generating the `requestId` for the request sequence.
+   *
+   * @default `nanoid`
+   */
+  idGenerator?: () => string
```

Comment:
Would like to propose alternative names such as `idFn`, `requestIdFn`, etc etc just make sure we're going with the best option. Any other suggestions?

Reply:
Eh, I'm find with `idGenerator`.

Reply:
As a consumer I'd be fine with all suggestions so far, so I'm pretty indifferent as to what it's called